build-logic-settings/settings.gradle.kts

```diff
@@ -22,7 +22,4 @@ dependencyResolutionManagement {
     }
 }
 
-include("allprojects-plugin")
```

Comment:
Can we keep one submodule here as well? It seems like we shouldn't have all the things in the root project, since then it is easier to split things out if we need to at some point.

Reply:
Done